jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/threadhistory/ThreadHistoryTagParser.java

```diff
@@ -0,0 +1,69 @@
+/*[INCLUDE-IF Sidecar18-SE]*/
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
```

Comment:
Some new files omit "and others"; please fix.

Reply:
Fixed.